src/cluster.h

```diff
@@ -4,58 +4,95 @@
 /*-----------------------------------------------------------------------------
  * Redis cluster data structures, defines, exported API.
  *----------------------------------------------------------------------------*/
-
+/* redis集群的总槽位数量 16384 */
 #define CLUSTER_SLOTS 16384
+/* 集群在线 */
 #define CLUSTER_OK 0            /* Everything looks ok */
+/* 集群挂了 */
 #define CLUSTER_FAIL 1          /* The cluster can't work */
+/* 集群节点名字长度 */
 #define CLUSTER_NAMELEN 40      /* sha1 hex length */
+/* 集群实际通信的端口 = 用户指定端口 + CLUSTER_PORT_INCR （6379 + 10000 = 16379）*/
 #define CLUSTER_PORT_INCR 10000 /* Cluster port = baseport + PORT_INCR */
 
 /* The following defines are amount of time, sometimes expressed as
  * multiplicators of the node timeout value (when ending with MULT). */
+/* 下面是和时间相关的一些常量，以_MULT结尾的常量会作为时间值的乘法因子来使用 */
+/* 节点故障报告的乘法因子 */
 #define CLUSTER_FAIL_REPORT_VALIDITY_MULT 2 /* Fail report validity. */
+/* 撤销主节点FAIL状态的乘法因子 */
```

Comment:
```suggestion
/* 撤销主节点 FAIL 状态的乘法因子 */
```
扫了一眼下边还有很多英文和中文之间没有空格的，需要加上空格来符合注释格式要求。

Reply:
请问一下这个注释审核容易过吗

Reply:
不会过不了的，只是可能有问题或者有要补充的会让你或者我们自己修改，然后你这个注释比较多所以可能会看得慢一点（主要是最近我也有别的事要做）

Reply:
好的